packages/bpk-docs/src/layouts/TokensLayout/TokensLayout.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import SideNavLayout from './../SideNavLayout';
+import * as routes from './../../constants/routes';
+
+const links = [
+  {
+    id: 'Tokens',
+    category: 'Tokens',
+    sort: true,
+    links: [
+      { id: 'COLORS', route: routes.COLORS, children: 'Colors' },
```

Comment:
It bugs me that these are not sorted alphabetically in the code. As this is an entirely new file can we at least do it here?

Reply:
If we sorted alphabetically in the code then there would be no use in the `sort: true` functionality - I dunno about you but when I add a new route I am too lazy to put it in alphabetical order ;)